src/sage/combinat/designs/incidence_structures.py

```diff
@@ -1049,14 +1050,25 @@ def is_simple(self) -> bool:
         B = self._blocks
         return all(B[i] != B[i + 1] for i in range(len(B) - 1))
 
-    def _gap_(self):
+    def _gap_init_(self):
         """
         Return the GAP string describing the design.
 
         EXAMPLES::
 
+            sage: # optional - gap_package_design
             sage: BD = IncidenceStructure(7,[[0,1,2],[0,3,4],[0,5,6],[1,3,5],[1,4,6],[2,3,6],[2,4,5]])
-            sage: BD._gap_()
+            sage: gap.load_package("design")
+            sage: gap(BD)
+            rec(
+              blocks := [ [ 1, 2, 3 ], [ 1, 4, 5 ], [ 1, 6, 7 ], [ 2, 4, 6 ],
+                  [ 2, 5, 7 ], [ 3, 4, 7 ], [ 3, 5, 6 ] ],
+              isBlockDesign := true,
+              v := 7 )
+
+        TESTS::
+
+            sage: BD._gap_init_()
```

Comment:
you need to copy here the optional tag and the definition of BD

Reply:
This bug is non-critical, (if I recalled correctly from the last time I work on this) there are too many code that relies on `x._gap_()` with no argument does the wrong thing (see the other test failures) and I just haven't gotten around to fix them yet.